src/processes.c

```diff
@@ -1529,8 +1529,14 @@ static char *ps_get_owner(pid_t pid)
         uid = atoi (line + 5);
         getpwuid_r (uid, &passwd, passwd_buffer, sizeof(passwd_buffer),
                 &passwd_result);
-        if (passwd_result)
+        if (passwd_result) {
             result = sstrdup (passwd_result->pw_name);
+        } else {
+	    // Send the numeric uid if name is not available.
+            char uid_str[6];  // 5 digits plus string terminator.
```

Comment:
Looked around online and information about uid_t does not seem to be current. So, assumed a worst case of 64 bits and adjusted the code accordingly.